docker-compose.yml

```diff
@@ -0,0 +1,29 @@
+version: '3'
+
+services:
+  zulip:
+    build:
+      context: .
+      dockerfile: Dockerfile-dev
+    ports:
+      - "9991:9991"
+    volumes:
+      - ".:/srv/zulip"
+    command: /srv/zulip/tools/start-compose
```

Comment:
Call this `start-docker-compose`, for clarity?  There's a lot of stuff in Zulip, and it seems worth being clear about what is what.